DEVELOPMENT.md

```diff
@@ -30,7 +30,7 @@ The docker environment described below should only be used for development and t
 
 To spin up a development environment for the *jenkins-datadog* plugin repository. The requirements are:
 
-* [Java 1.8](https://www.java.com/en/download/)
+* [Java 1.11](https://www.java.com/en/download/)
```

Comment:
```suggestion
* [Java 11](https://www.java.com/en/download/)
```
Starting with JDK 9 the correct way to specify release is without the `1.` prefix